yroots/subdivision.py

```diff
@@ -99,61 +99,60 @@ def solve(funcs, a, b, rel_approx_tol=1.e-15, abs_approx_tol=1.e-12,
     -------
     zeros : numpy array
         The common zeros of the polynomials. Each row is a root.
-    '''
-    #Detect the dimension
+    """
+    # Detect the dimension
     if not isinstance(funcs,list):
         dim = 1
     else:
         dim = len(funcs)
 
-    #make a and b the right type
+    # make a and b the right type
     a = np.float64(a)
     b = np.float64(b)
 
     # Choose an appropriate max degree for the given dimension if none is specified.
     if deg is None:
-        deg_dim = {1: 50, 2:9, 3:5, 4:3}
+        deg_dim = {1: 100, 2:20, 3:9, 4:9}
         if dim > 4:
             deg = 2
```

Comment:
Any reason why we're leaving this at 2? I guess we can come back to this later as we expand into higher dimensions.

Reply:
We haven't run experiments to determine whether 2 would be the best degree in higher dimensions, so I just kept this as the default from previous versions.